draft-ietf-oauth-v2-1.md

```diff
@@ -1030,8 +965,53 @@ Client authentication is used for:
    of refresh tokens can be challenging, while rotation of a single
    set of client credentials is significantly easier.
 
+## Token Request {#token-request}
+
+The client makes a request to the token endpoint by sending the
+following parameters using the `application/x-www-form-urlencoded`
+format per Appendix B with a character encoding of UTF-8 in the HTTP
+request payload:
+
+"client_id":
+:    REQUIRED, if the client is not authenticating with the
+     authorization server as described in {{token-endpoint-client-authentication}}.
+
+"scope":
+:    OPTIONAL.  The scope of the access request as described by
+     {{access-token-scope}}.
+
+"grant_type":
```

Comment:
"client_id" and "scope" are the common denominator among all grant types whereas "grant_type" is the switch between the different grant types. That's why I put it here.

Reply:
That's actually exactly why I would expect to see it first in the list, since it's the thing that changes the expected outcome of the request. In any case we can change that back later, I don't want this to hold up the PR.